docs/user_guide/sd_robot_description.rst

```diff
@@ -32,15 +32,17 @@ Current allowed configurations are the following:
 |              | th,ff,mf,rf,lf |     th,ff,mf,rf     |          th,ff,mf         |      th,ff,mf,rf,lf      |
 +--------------+----------------+---------------------+---------------------------+--------------------------+
 
-There are also arguments that define where and which sensors are located on the hand. It allows placement of sensors on tip, mid and proximal parts of the fingers as well as the palm. Argument names: ``tip_sensors``, ``mid_sensors``, ``prox_sensors``, ``palm_sensor``. Currently, only sensors at the fingertips are available. There are three fingertip sensor types: ``pst``/``bt_sp``/``bt_2p``.
-
-+-------------+-----+------------------+
-|             | PST | Syntouch Biotacs |
-|             |     +---------+--------+
-|             |     |    2p   |   sp   |
-+=============+=====+=========+========+
-| tip_sensors | pst |  bt_2p  |  bt_sp |
-+-------------+-----+---------+--------+
+There are also arguments that define where and which sensors are located on the hand. It allows placement of sensors on tip, mid and proximal parts of the fingers as well as the palm. Argument names: ``tip_sensors``, ``mid_sensors``, ``prox_sensors``, ``palm_sensor``. Currently, only sensors at the fingertips are available. There are three fingertip sensor types: ``mst``/``pst``/``bt_sp``/``bt_2p``.
```

Comment:
Should we clarify why it says "three" types but lists four. (I.e. are two types of biotacs only one type of sensor? maybe that's fine but we should probably state it.)

Reply:
You're right, let's just say we support 4: [3d1137c](https://github.com/shadow-robot/sr_dexterous_hand_documentation/pull/180/commits/3d1137c5ffa6387f4f4e1d55307dacc61a325132)